app/plugin_api.go

```diff
@@ -330,6 +330,10 @@ func (api *PluginAPI) GetChannel(channelId string) (*model.Channel, *model.AppEr
 	return api.app.GetChannel(channelId)
 }
 
+func (api *PluginAPI) GetChannels(options *model.GetChannelsOptions) (*model.ChannelList, *model.AppError) {
```

Comment:
Is there a need for returning `*model.ChannelList`? Why not just `[]*model.Channel`?